c++ training/PointersTraining/PointerExercises/Project1/Source.cpp

```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+
+/*	Exercise 1 - Understanding Memory Addresses
+Create a program that will ask the user to enter an integer and assign the given value to a
+variable.Then, print the value to the console, and print the memory address of that variable.
+Then, ask for another valueand assign that value to the same variable.Print the valueand the
+memory address to the console.
+Finally, ask for one more value.This time, assign the value to a different variable.Print the
+valueand the memory address to the console.Do the results make sense to you ?*/
+
+int main()
+{
+	std::cout << "Please enter a number: ";
+
+	int IntVariable;
+	std::cin >> IntVariable;
```

Comment:
It can be annoying, but it is better to get used to using validation and inputs always